modules/ap.sh

```diff
@@ -8,6 +8,11 @@ function ap {
   channels=(1 6 11)
   channel=${channels[$((RANDOM % ${#channels[@]}))]};
 
+  if [[ "${#essid}" -gt 32 && "$essid" = *" "* ]]; then
+    echo "Error: ssid must have less than 32 character and no spaces"
```

Comment:
`echo "Error: ssid must have less than or equal to 32 characters and no spaces"`

or

`echo "Error: ssid cannot have more than 32 characters and no spaces"`

Reply:
A more serious issue:
Running `./cli.sh ap internet "tree houses"` (so using an essid with a space and no password)  seems to crash things.

Reply:
> A more serious issue:
> Running `./cli.sh ap internet "tree houses"` (so using an essid with a space and no password) seems to crash things.

I got this problem also.

Reply:
I wrote a wrong if statement. :)